infra/images/src/main/java/com/simprints/infra/images/metadata/database/DbImageMetadata.kt

```diff
@@ -0,0 +1,15 @@
+package com.simprints.infra.images.metadata.database
+
+import androidx.room.Entity
+import com.google.errorprone.annotations.Keep
+
+@Entity(
+    tableName = "DbImageMetadata",
+    primaryKeys = ["imageKey", "key"],
+)
+@Keep
+internal data class DbImageMetadata(
+    val imageKey: String,
```

Comment:
I would prefer to name it imageId instead of imageKey to avoid any ambiguity with the metadata key